starter-code/src/app/movies.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { Movie, movies } from './movies-data';
+
+@Injectable()
+export class MoviesService {
+
+  movies: Movie[] = [...movies];
```

Comment:
Is there a reason we are creating a copy of the array here?
Instead of `"movies: Movie[] = movies"`